draft-ietf-quic-v2.md

```diff
@@ -139,13 +139,18 @@ nonce = 0x141b99c239b03e785d6a2e9f
 
 # Version Negotiation Considerations
 
-QUIC version 2 endpoints SHOULD also support QUIC version 1. Any QUIC endpoint
-that supports multiple versions MUST meet the minimum requirements described in
-{{QUIC-VN}} to prevent version downgrade attacks.
+QUIC version 2 endpoints SHOULD also support QUIC version 1, as version 2 is
+both less common than, and not intended to deprecate, version 1. Any QUIC
```

Comment:
I don't think saying anything about less common makes sense, we can't predict the future. This requirement comes from the fact that we kept the same Alt-Svc, so it makes sense to scope it to h3?